guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -174,32 +245,72 @@ Incremental upgrade (of a content view):: The act of creating a new (minor) cont
 Incremental upgrades provide a way to make in-place modification of an already published content view.
 Useful for rapid updates, for example when applying security errata.
 
+[[Installation_Media]]
+Installation Media::
+Installation media are packages used to install the base operating system during the provisioning process.
+An installation medium in {Project} represents the installation files for one or more operating systems, which must be accessible over the network, either through an URL or an NFS server location.
+It is usually either a mirror or a CD or DVD image.
+// would you want to do this for RHEL or rather use a synchronized Kickstart repository?
+Pointing the URL of the installation medium to a local copy, for example `http://{foreman-example-com}/pub/installation_media/`, can improve provisioning time and reduce network load.
++
+Every operating system depends on exactly one path of an installation medium, whereas installation media paths may serve different operating systems at the same time.
+ifdef::satellite[]
+You
+endif::[]
+ifndef::satellite[]
+For {EL}, you
+endif::[]
+can use `$version`, `$major`, and `$minor` to parameterize the URL.
+ifndef::satellite[]
+Debian, Ubuntu, and {SLES} do not support such variables.
++
+You can bundle installation media for multiple versions of Debian or Ubuntu in one path, for example a repository containing packages for both _Debian 11 "Bullseye"_ and _Debian 12 "Bookworm"_.
+endif::[]
+
 [[Job]]
 Job:: A command executed remotely on a host from {ProjectServer}.
 Every job is defined in a job template.
 
 [[Katello]]
-Katello:: A Foreman plugin responsible for subscription and repository management.
+Katello::
+An open source Foreman plugin to perform lifecycle management of registered hosts, content management, and subscription handling.
+It depends on Pulp for content management, which fetches software from repositories and stores various versions of it.
```

Comment:
It also depends on Candlepin for host registration and managing subscription manifests :)

Reply:
Added.